package.json

```diff
@@ -71,7 +71,8 @@
   },
   "nyc": {
     "exclude": [
-      "**/*.test.js"
+      "**/*.test.js",
+      "tests/fixtures.js"
```

Comment:
Would like to exclude the absolutePath more granularly (mocking process.platform is a bit overkill for this scenario i thought), but haven't found a way to do that.